lib/core/src/server/build-static.js

```diff
@@ -192,7 +192,12 @@ export async function buildStaticStandalone(options) {
   shelljs.cp('-r', dllPath, path.join(outputDir, 'sb_dll'));
 
   await buildManager(configType, outputDir, configDir, options);
```

Comment:
does buildManager re-use prebuild / cache?

should we add this? if people don't add custom addons, this has the potential to make publishing storybook a lot faster!

Reply:
maybe we can add a --prebuild-manager="{{path_to_prebuilt}}", so users can pull their custom prebuilt manager from some directory?